tools/logrotate_config/package_after_install_script.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
rename the directory to log_rotate/install_script.sh

Reply:
done.